packages/schematics/src/collection/workspace/index.ts

```diff
@@ -220,6 +220,9 @@ function updateTsLintJson(options: Schema) {
       ].forEach(key => {
         json[key] = undefined;
       });
+      if(!json.rulesDirectory) {
```

Comment:
Sorry for being a little nitpicky here. Can you change it to a fallback assignment?

```ts
json.rulesDirectory = json.rulesDirectory || [];
```

Also can you add a test in https://github.com/nrwl/nx/blob/master/packages/schematics/src/collection/workspace/workspace.spec.ts

Let me know if you need help.

Reply:
No problem on the nitpick. That's definitely better!

For the test, I'm not really sure how to do this without building an entire app tree. None of the other workspace tests actually successfully run the command - they just test for exceptions being thrown.

Am I missing something?

Reply:
Okay that's fine for now then just make the slight fix